src/services/apimPolicyBuilder.ts

```diff
@@ -80,19 +82,24 @@ export class ApimPolicyBuilder {
   public ipFilter(ipConfig: ApiIpFilterPolicy): ApimPolicyBuilder {
     Guard.null(ipConfig, "ipConfig");
 
-    const element = {
+    const policy = {
       $: { action: ipConfig.action },
```

Comment:
I have not seen `$` being used as key before.  Is there some benefits to this?

Reply:
`$` is an artifact of the JSON -> XML library we use.  In this context `$` gets access to the attributes applied to an element.